<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageForgeSettingsGeneral</class>
 <widget class="QWidget" name="PageForgeSettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="labelEditModeAccess">
     <property name="text">
      <string>Edit Mode Access</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer_16">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>397</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="fieldEditorRespawnTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldGlobalVoice">
     <property name="text">
      <string>Enable Open-Channel Voice</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldFlag1">
     <property name="text">
      <string>Unknown Flag 1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEditorRespawnTime">
     <property name="text">
      <string>Editor Respawn Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="fieldEditModeAccess">
     <item>
      <property name="text">
       <string>unknown 0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unknown 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unknown 2</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
